internal/client.go

```diff
@@ -643,9 +644,20 @@ type (
 		// Optional: defaulted to Fail.
 		WorkflowIDConflictPolicy enumspb.WorkflowIdConflictPolicy
 
+		// WithStartWorkflowOperation - Operation to execute with Workflow Start.
+		// For example, see NewUpdateWorkflowOperation to perform Update-with-Start. Note that if the workflow is
+		// already running and WorkflowIDConflictPolicy is set to UseExisting, the start is skipped and only the
+		// operation is executed. If instead the policy is set to Fail (the default), nothing is executed and
+		// an error will be returned (i.e. the option WorkflowExecutionErrorWhenAlreadyStarted is ignored).
+		//
+		// Optional: defaults to nil.
+		WithStartWorkflowOperation WithStartWorkflowOperation
```

Comment:
What happens if this is set when a user calls `SignalWithStartWorkflow`?

Reply:
It will be ignored. Would a comment on `SignalWithStartWorkflow` or `WithStartOperation` (or both) be best here?

Reply:
I think we should document it in `WithStartOperation` and error if it is set when calling `SignalWithStartWorkflow`. You can also document it in `SignalWithStartWorkflow` but I don't think that is required